integrations/tensorflow/CMakeLists.txt

```diff
@@ -18,9 +18,8 @@ if(${IREE_BUILD_PYTHON_BINDINGS})
 endif()
 
 if(${IREE_BUILD_TESTS} AND ${IREE_BUILD_PYTHON_BINDINGS})
-  if(${IREE_ENABLE_NEW_INTEGRATION_TESTS})
-    add_subdirectory(test)
-  else()
+  add_subdirectory(test)
+  if(NOT ${IREE_ENABLE_NEW_INTEGRATION_TESTS})
```

Comment:
This condition is confusing to me. The variable name does not appear to reflect what it's being used for.

Reply:
Updated the flags to be more clear. Each set can be separately enabled/disabled.